openadapt/tools/printer.py

```diff
@@ -0,0 +1,98 @@
+import subprocess
+import platform
+
+def get_available_printers():
```

Comment:
Can you please run `black` on this PR, and set up your pre-commit hooks as described in the README so it happens automatically in the future? 

(There should be two newlines before a function definition)

Reply:
addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/359/commits/e0a82e421de144cf9f9f6b0258943bc929dc821f